src/shared/dropdown-wrapper/element-with-dropdown.tsx

```diff
@@ -77,6 +79,20 @@ export const ElementWithDropdown = ({
         targetRef: elementRef,
         handleHeight: false,
     });
+    const center: Middleware = {
+        name: "center",
+        fn: ({ x, rects }) => {
+            const noGapInBetween =
```

Comment:
also consider the case where dropdown is right aligned. `x` will be 0

Reply:
updated, add in x===0